release.yaml

```diff
@@ -4,6 +4,10 @@ fleet-agent:
   - 103.1.1+up0.9.1-rc.5
 fleet-crd:
   - 103.1.1+up0.9.1-rc.5
+harvester-cloud-provider:
+  - 103.0.1+up0.2.3
+harvester-csi-driver:
+  - 103.0.1+up0.1.17
```

Comment:
This PR shouldn't be introducing these charts, you can probably remove them unless the validation fails without

Reply:
Looks like someone may have been pushing directly to dev-v2.8? because these versions have been mysteriously appearing and disappearing for me in some of my branches when I rebase. I appended them back to the end of the file since they are now in the upstream dev-v2.8